provision-horde.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/sh
+
+# shellcheck disable=1091
+. mail-toaster.sh || exit
+
+export JAIL_START_EXTRA=""
+# shellcheck disable=2016
+export JAIL_CONF_EXTRA=""
+export JAIL_CONF_EXTRA="
+        mount += \"$ZFS_DATA_MNT/horde \$path/data nullfs rw 0 0\";
+        mount += \"$ZFS_DATA_MNT/vpopmail \$path/usr/local/vpopmail nullfs rw 0 0\";"
+
+mt6-include 'php'
+mt6-include nginx
+
+install_horde()
+{
+
+    if [ ! -d "$ZFS_DATA_MNT/horde/data" ]; then
+        tell_status "creating $ZFS_DATA_MNT/horde/data"
+        mkdir "$ZFS_DATA_MNT/horde/data"
+    fi
+
+    tell_status "making vpopmail dir"
+    mkdir -p "$STAGE_MNT/usr/local/vpopmail"  || exit
+
+    install_php 56 || exit
+	install_nginx || exit
```

Comment:
looks like you've got a mix of tabs and spaces. Usually you can configure your editor to automatically create consistent whitespace within files.